front_end/helper.py

```diff
@@ -83,16 +83,16 @@ def get_columns_dict(nested_list, key_col_index, value_col_index):
 def exec_code(settings_dict, code, problem_basics, problem_details, request=None):
     code = code + "\n\n" + problem_details["test_code"]
 
-    if request:
-        for url, file_name in get_columns_dict(problem_details["data_urls_info"], 0, 1).items():
-            cache_url = "{}://{}/data/{}/{}/{}/{}".format(
-                request.protocol,
-                request.host,
-                problem_basics["assignment"]["course"]["id"],
-                problem_basics["assignment"]["id"],
-                problem_basics["id"],
-                file_name)
-            code = code.replace(url, cache_url)
+    # if request:
```

Comment:
We need to keep this logic in there. Just simplify it a bit because there will only be a max of one data URL.